CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing
+
+Thanks for contributing! Our community of developers is what put SmartThings on the map!
+
+## How can I contribute?
+
+### Improve documentation
+
+As a user of the SmartThings CLI, you're the perfect candidate to help us improve our documentation. Error fixes, typo corrections, better explanations, more examples, etc. Open issues for things that could be improved – anything. Even improvements to this document.
+
+### Give feedback on issues
+
+We're always looking for more opinions on discussions in the issue tracker. It's a good opportunity to influence the future direction of the CLI.
+
+### Submitting an issue or feature request
+
+- Search the issue tracker before opening an issue
+- Ensure you're using the latest version
+- Use a clear and descriptive title
+- Include as much information as possible by filling out the issue template
+- The more time you put into an issue, the more we will
+
+### Submitting a pull request
+
+- Non-trivial changes are often best discussed in an issue first, to prevent you from doing unnecessary work.
+- For ambitious tasks, you should try to get your work in front of the community for feedback as soon as possible. Open a pull request as soon as you have done the minimum needed to demonstrate your idea. At this early stage, don't worry about making things perfect, or 100% complete. Add a `[WIP]` prefix to the title, and describe what you still need to do. This lets reviewers know not to nit-pick small details or point out improvements you already know you need to make.
```

Comment:
Should we suggest draft mode instead since that should be available now?

Reply:
Yeah, good idea thanks.